v03_pipeline/lib/annotations/snv_indel.py

```diff
@@ -7,38 +8,6 @@
 N_ALT_HOM = 2
 
 
-def _AC(row: hl.StructExpression) -> hl.Int32Expression:  # noqa: N802
```

Comment:
These functions were originally split out as separate annotations.  Good for modularity but bad for performance.